lib/morse_translator.rb

```diff
@@ -40,4 +40,22 @@ def initialize
                     " " => " "}
   end
 
+  def translate(english)
+    english.downcase.split("").map { |letter| @dictionary[letter]}.join
```

Comment:
This line of code is doing way to much. The single line syntax is not appropriate in this case. you should use do/end